tests/attribute_annotations.pyi

```diff
@@ -13,4 +13,4 @@ class Foo:
     field4: int = 0  # Y015 Attribute must not have a default value other than "..."
     field5 = 0  # type: int  # Y015 Attribute must not have a default value other than "..."
     field6 = 0  # Y015 Attribute must not have a default value other than "..."
-    field7 = ""  # Y015 Attribute must not have a default value other than "..."
+    field7 = b""  # Y015 Attribute must not have a default value other than "..."
```

Comment:
- If it was left to just `""`, would it generate two errors, one because it is a string and another because it is a default value?
- It would be good to test what happens to docstrings. The AST that docstrings create depends on the Python version, if I remember correctly: on some Python versions the docstring is a statement in the class or function, on others it is in a separate attribute of the class/function definition object.

Reply:
Yes, it would generate two errors.

I'll add tests for docstrings. I believe the AST doesn't actually depend on Python version—they were going to change it but then backed it out because it was too disruptive. In any case it's useful to test.

Reply:
FYI in case we need it later, it wouldn't be hard to make the test suite support multiple errors on a single line by specifying multiple error comments: basically change `re.search` to `re.findall`, and replace `.*` with `[^\n#]*`.

Reply:
I just implemented that before I saw your comment :)